fastlane/lanes/localization.rb

```diff
@@ -229,9 +225,9 @@
       download_path: download_path
     )
 
-    # TODO: Remove this call after a few sprints of validating that that the `skip_upload_metadata: options[:track] != 'production'` recently added to `upload_build_to_play_store` lane works well
-    android_create_xml_release_notes(download_path: download_path, build_number: "#{options[:build_number]}", locales: RELEASE_NOTES_LOCALES)
-    sh("git add #{download_path} && git commit -m \"Update WordPress metadata translations for #{options[:version]}\" && git push origin HEAD")
+    git_add(path: download_path)
+    git_commit(message: "Update WordPress metadata translations for #{options[:version]}", allow_nothing_to_commit: true)
```

Comment:
```suggestion
    git_commit(path: download_path, message: "Update WordPress metadata translations for #{options[:version]}", allow_nothing_to_commit: true)
```

I tried this locally and it asked me for input, which I don't think it's what we want here.

<img width="1497" alt="Screen Shot 2022-04-11 at 3 32 09 pm" src="https://user-images.githubusercontent.com/1218433/162671300-ff16b311-9136-4735-bcfc-6f3674c79d4c.png">

Reply:
I wanted to make sure my suggestion was correct so I tested it locally, might as well open a PR for it while at it: https://github.com/wordpress-mobile/WordPress-Android/pull/16290.

Reply:
Addressed by merging your PR #16290 into this branch 🙂 🙇